Protos/api.proto

```diff
@@ -87,5 +87,10 @@ message TaskDataParam {
     oneof dataParam {
         Future future = 1;
         bytes data = 2;
+        FutureList futures = 3;
```

Comment:
It needs to have both Future and FutureList?

Reply:
one of them

Reply:
Future is kept for backward compatibility